#include <ooxi/exception/Exception.h++>

int main() {
	THROW(MyException, "This is my exception");
}
